app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/ThreadActionsBottomSheetDialog.kt

```diff
@@ -88,10 +93,10 @@ class ThreadActionsBottomSheetDialog : MailActionsBottomSheetDialog() {
             setSnoozeUi(thread.isSnoozed())
         }
 
-        getThreadAndMessageUidToReplyTo().observe(viewLifecycleOwner) { result ->
-            result?.let { (thread, messageUidToReply) ->
-                setupListeners(thread, messageUidToReply)
-            } ?: findNavController().popBackStack()
+        getThreadAndMessageUidToReplyTo().observe(viewLifecycleOwner) { junkMessageThreadData ->
+            val firstJunkMessageThreadData = junkMessageThreadData.first()
```

Comment:
If the data is empty, first will crash with an exception, use firstOrNull() then keep the popbackStack in case of error